src/debugmode.lua

```diff
@@ -73,12 +73,18 @@ function Debug:draw()
 		end
 	end
 
+	local chartWidth = #drawTimes
```

Comment:
drawTimes is still a global variable; it's on my to do list to go back and fix that.